frontend/src/pages/Nodes/NodeVoting.js

```diff
@@ -131,10 +132,12 @@ const NodeVoting = ({
     approveNewNodeForExistingOrganization(address)
```

Comment:
I am a little bit confused maybe due to variable names. Logically the code has to do following things if I understand it right:
- split nodes into `newOrgaNodes` and `existingOrgaNodes`
- for each of these 2 types of nodes a listentry has to be rendered
- In case of newOrgaNodes approveNewOrganization is called on approve and EmptyState.newOrgaNodes should be rendered if the table has no entries
- In case of existingOrgaNodes approveNewNodeForExistingOrganization is called on approve and EmptyState.existingOrgaNodes should be rendered if the table has no entries

I would expect something like this:
```
const newOrgaNodes, existingOrgaNodes  = splitNodes(nodes)
newOrgaNodesListItems = getListEntries(newOrganizationNodes, canApprove, classes, ({ organization }) =>
    approveNewOrganization(organization))
// in JSX
newOrgaNodesListItems.length > 0 ?  newOrgaNodesListItems : EmptyStates.newOrgaNodes
```